--- folks-0.12.1/backends/eds/meson.build.orig	2019-04-24 13:20:07.000000000 +0200
+++ folks-0.12.1/backends/eds/meson.build	2019-05-19 19:58:55.357659020 +0200
@@ -25,7 +25,7 @@
   '-DG_LOG_DOMAIN="@0@"'.format(eds_backend_name),
 ]
 
-eds_backend = library('eds',
+eds_backend = shared_library('eds',
   eds_backend_sources,
   dependencies: eds_backend_deps,
   vala_args: eds_backend_vala_flags,
